docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -49,105 +49,1378 @@ SETTINGS cast_keep_nullable = 1
 └──────────────────┴─────────────────────┴──────────────────┘
 ```
 
-## toInt(8\|16\|32\|64\|128\|256)
+## toInt8
 
-Converts an input value to a value the [Int](../data-types/int-uint.md) data type. This function family includes:
+Converts an input value to a value of type [`Int8`](../data-types/int-uint.md). Throws an exception in case of an error.
 
-- `toInt8(expr)` — Converts to a value of data type `Int8`.
-- `toInt16(expr)` — Converts to a value of data type `Int16`.
-- `toInt32(expr)` — Converts to a value of data type `Int32`.
-- `toInt64(expr)` — Converts to a value of data type `Int64`.
-- `toInt128(expr)` — Converts to a value of data type `Int128`.
-- `toInt256(expr)` — Converts to a value of data type `Int256`.
+**Syntax**
+
+```sql
+toInt8(expr)
+```
+
+**Arguments**
+
+- `expr` — Expression returning a number or a string representation of a number. [Expression](../syntax.md/#syntax-expressions).
+
+Supported types:
+- (U)Int8/16/32/64/128/256
+- Float*
```

Comment:
`Float*` was my idea but in the interest of readability, `Float32/64` would be better (sorry for the confusion).